torchao/sparsity/prototype/training/README.md

```diff
@@ -0,0 +1,53 @@
+# Accelerated Sparse Training
```

Comment:
why prototype? The API seems quite nice already if it module swaps on linear should be able to support most interesting models?

Reply:
It's not an issue